bepasty/utils/permissions.py

```diff
@@ -54,7 +54,7 @@ def get_permissions():
 def get_permission_icons():
     return [
         (permission, permission_icons[permission])
-        for permission in get_permissions()
+        for permission in get_permissions() if permission
```

Comment:
feel free to try it yourself, maybe my setup was just misconfigured or smth like that